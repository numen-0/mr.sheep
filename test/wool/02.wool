main:
    mov &0, 1
    mov ^0, 'H'  inc &0
    mov ^0, 'e'  inc &0
    mov ^0, 'l'  inc &0
    mov ^0, 'l'  inc &0
    mov ^0, 'o'  inc &0
    mov ^0, ' '  inc &0
    mov ^0, 'W'  inc &0
    mov ^0, 'o'  inc &0
    mov ^0, 'r'  inc &0
    mov ^0, 'l'  inc &0
    mov ^0, 'd'  inc &0
    mov ^0, '\n' inc &0
    mov ^0, 0
    mov &0, 1
loop:
    print_c ^0
    inc &0
    JIT ^0, loop
end:
    EXIT 0
