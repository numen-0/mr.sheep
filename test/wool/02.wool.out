mr.sheep: successfully loaded './test/wool/02.baa' of 78B
Hello World

mr.sheep: program exited with code: 0
